lib/widgets/screens/inbox/inbox_chats.dart

```diff
@@ -84,15 +70,40 @@ class _InboxChatsScreenState extends State<InboxChatsScreen>
             });
           },
           icon: Icons.archive_outlined,
-          child: InboxListTile(
-            avatarUrl: channelAvatarUrl,
-            fullName: channelName,
-            date: DateTime.now().subtract(Duration(
-                days: i)), // TODO(m-rosario): Use date from last message
-            message: '', // TODO(m-rosario): Get text from last message
-            notifications: channelUnseenMessages.length,
-            onPressed: () =>
-                context.push('${Routes.inboxChats.path}/${channel.id}'),
+          child: messagesProvider.findChannelMessages(
+            input: Input$ChannelMessageListFilter(channelId: channel.id),
```

Comment:
Sent this to @m-rosario in chat but moving discussion here for visibility. 

This request should get only the text from the last message and the channel ID. 

```graphql
query FindChannelMessages($filter: ChannelMessageListFilter, $options: FindObjectsOptions) {
  findChannelMessages(filter: $filter, options: $options) {
    channelId
    messageText
  }
}
```
```json
{
  "filter": {
    "channelId": "647df8b494e2b5c62724c276"
  },
  "options": {
    "limit": 1,
    "sort": [
      {
        "direction": "desc",
        "field": "createdAt"
      }
    ]
  }
}
```

Reply:
@andersan I tried to implement this, but the schema defines the 'sort' field as a String, meaning that I cannot provide the options that you're giving me here. Is there a newer version of the schema?

Reply:
@m-rosario yes, the newer version is in #73. Didn't realize that the schema had changed here, Holger implemented this a few weeks ago.

I could merge #73 into main and finish off the content provider later if that'd be helpful.

Reply:
@andersan Don't worry about it I will just update the query once it's merged. I was just curious why it didn't work